spec/spec_helper.rb

```diff
@@ -11,6 +11,20 @@
 
   config.shared_context_metadata_behavior = :apply_to_host_groups
 
+  require "webmock/rspec"
+  WebMock.disable_net_connect!(allow_localhost: true)
+
+  config.before(:each) do
+    stub_request(:any, "https://brianiswu-open-brewery-db-v1.p.rapidapi.com/breweries/search").
```

Comment:
Ignoring this for now, because I am getting conflicting messages from Hound and Rubocop about `Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.` and `Layout/DotPosition: Place the . on the next line, together with the method name.`. I'm choosing to ignor `Metrics/LineLength` here for readability.